python/ql/src/semmle/python/types/Exceptions.qll

```diff
@@ -276,7 +276,7 @@ class ExceptFlowNode extends ControlFlowNode {
             or
             exists(TupleValue tup |
                 this.handledObject(tup, ClassValue::tuple(), _) |
-                val = tup.getItem(_) and origin.pointsTo(val)
+                element_from_tuple(tup).pointsTo(val,origin) 
```

Comment:
Why re-add `element_from_tuple` when you could just use `origin = val.getOrigin()`?

Reply:
Ah, yes, right, somehow I got lost in changes that cascaded from `getOrigin` heh